tests/StatusCommandTest.php

```diff
@@ -40,14 +41,64 @@ public function it_checks_if_local_translations_match_downloaded_translations()
         $this->artisan('poeditor:status')
             ->expectsOutput('The translations for \'en\' do not match the ones on POEditor.')
             ->doesntExpectOutput('The translations for \'nl\' do not match the ones on POEditor.')
+            ->expectsTable(
+                ['Translation Key'],
+                [
+                    ['php-file.baz'],
+                ]
+            )
             ->assertExitCode(1);
 
-        $this->createPhpTranslationFile('en/php-file.php', ['foo' => 'bar', 'nested' => ['value' => 'nested value']]);
+        $this->createPhpTranslationFile('en/php-file.php', ['foo' => 'bar', 'nested' => ['value' => 'nested value'], 'baz' => 'baz', 'foo' => 'bar']);
 
         $this->artisan('poeditor:status')
             ->expectsOutput('All translations match the ones on POEditor!')
             ->doesntExpectOutput('The translations for \'en\' do not match the ones on POEditor.')
             ->doesntExpectOutput('The translations for \'nl\' do not match the ones on POEditor.')
             ->assertExitCode(0);
     }
+
+    /** @test */
+    public function it_does_not_fail_if_translations_in_different_order()
+    {
+        config()->set('poeditor-sync.locales', ['en']);
+
+        $this->createPhpTranslationFile('en/php-file.php', ['foo' => 'bar', 'baz' => 'bar']);
+
+        $this->mockPoeditorDownload('en', [
+            'php-file' => [
+                'bar' => 'baz',
+                'foo' => 'bar',
+            ],
+        ]);
+
+        $this->artisan('poeditor:status')
+            ->expectsOutput('The translations for \'en\' do not match the ones on POEditor.')
```

Comment:
test name zegt "it does not fail" maar resultaat is wel een failure;

in mockPoeditorDownload moet `'bar' => 'baz',` vervangen worden door `'baz' => 'bar'` en dan zou command wel moeten slagen (ook `->expectsOutput('All translations match the ones on POEditor!')` toevoegen dan)